ksql-parser/src/main/java/io/confluent/ksql/parser/tree/CreateStream.java

```diff
@@ -44,10 +44,20 @@ public CreateStream(
     super(location, name, elements, notExists, properties);
   }
 
+  private CreateStream(
```

Comment:
likewise, can we not drop the old constructor?

Reply:
They're being used all over the code for testing, and I think it's a useful utility method to have.